src/apps/dashboard/dashboard-fees/dashboard-fees.tsx

```diff
@@ -12,11 +12,11 @@ const DashboardFees: FC = () => {
 
   const feesPageUrl = u("feesPageUrl");
   const { data: fbsFees } = useGetFeesV2();
-  const [feeCount, setFeeCount] = useState<number>();
+  const [feeCount, setFeeCount] = useState<number>(0);
   const [totalFeeAmount, setTotalFeeAmount] = useState<string>("0");
 
   useEffect(() => {
-    if (fbsFees) {
+    if (fbsFees && Array.isArray(fbsFees)) {
```

Comment:
I get the point and it makes sense. The things is that maybe, in this case, we actually want some empty views on the various pages. Throwing errors for unregistered users introduces that we also need to handle those errors instead of showing an error boundary.